x/auth/changepubkey/handler.go

```diff
@@ -0,0 +1,51 @@
+package changepubkey
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	changepubkeykeeper "github.com/cosmos/cosmos-sdk/x/auth/changepubkey/keeper"
+	"github.com/cosmos/cosmos-sdk/x/auth/changepubkey/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/keeper"
+)
+
+// NewHandler returns a handler for x/auth message types.
+func NewHandler(ak keeper.AccountKeeper, pk changepubkeykeeper.ChangePubKeyKeeper) sdk.Handler {
+	return func(ctx sdk.Context, msg sdk.Msg) (*sdk.Result, error) {
+		ctx = ctx.WithEventManager(sdk.NewEventManager())
+
+		switch msg := msg.(type) {
+		case *types.MsgChangePubKey:
+			return handleMsgChangePubKey(ctx, ak, pk, msg)
+
+		default:
+			return nil, sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "unrecognized %s message type: %T", types.ModuleName, msg)
+		}
+	}
+}
+
+func handleMsgChangePubKey(ctx sdk.Context, ak keeper.AccountKeeper, pk changepubkeykeeper.ChangePubKeyKeeper, msg *types.MsgChangePubKey) (*sdk.Result, error) {
+	acc := ak.GetAccount(ctx, msg.Address)
```

Comment:
Why is all this logic not in a keeper method?

Reply:
Figured it wasn't worth creating a keeper, because it doesn't use its own store or anything, it just does a simple operation on the AccountKeeper.  Is it worth having a whole keeper for that?